packages/venia-ui/lib/components/MiniCart2/minicart.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { bool, func, shape, string } from 'prop-types';
+
+import { useScrollLock } from '@magento/peregrine';
+import { useMiniCart2 } from '@magento/peregrine/lib/talons/MiniCart2/useMiniCart2';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import defaultClasses from './minicart.css';
+
+/**
+ * The MiniCart component shows a small overview of the user's
+ * cart on desktop only.
+ *
+ * @param {*} props
+ */
+const MiniCart = props => {
+    const { isOpen, setIsOpen } = props;
+
+    // Prevent the page from scrolling in the background
+    // when the MiniCart is open.
+    useScrollLock(isOpen);
+
+    const talonProps = useMiniCart2({
+        setIsOpen
+    });
+
+    const { onDismiss } = talonProps;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const rootClass = isOpen ? classes.root_open : classes.root;
+
+    return (
+        <aside className={rootClass}>
+            {/* The Mask. */}
+            <button className={classes.mask} onClick={onDismiss} type="reset" />
+            {/* The Contents. */}
+            <div className={classes.contents}>
+                <div className={classes.header}>Header TBD</div>
+                <div className={classes.body}>
+                    Items List TBD <br />
```

Comment:
```suggestion
                    {Array(20).fill(<div>Items List TBD</div>)}
```

:D

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2494/commits/18d615ed3d7638d1cf9e800b5ea9a586c08601e3.